core/dbt/cli/params.py

```diff
@@ -172,7 +172,12 @@
 )
 
 output_keys = click.option(
-    "--output-keys", envvar=None, help="TODO: No current help text", type=click.STRING
+    "--output-keys",
+    envvar=None,
+    help="TODO: No current help text",
```

Comment:
@dbeatty10 Could I ask for your opinion on some help text here as well please 😅

Reply:
See below on some ideas we can iterate on inspired by the docs on [listing JSON output with custom keys](https://docs.getdbt.com/reference/commands/list#:~:text=Listing%20JSON%20output%20with%20custom%20key):

Idea 1:
```
Space-delimited listing of which node properties are included in the output when `--output json`.
```

Idea 2:
```
Custom keys for JSON output when `--output json`. Space-delimited listing of node properties to include.
```

Idea 3:
```
Space-delimited listing of node properties to include as custom keys for JSON output.
```

If we include examples in the help text, then maybe something like this:
```
Space-delimited listing of node properties to include as custom keys for JSON output.
Ex. `--output json --output-keys "name resource_type description"`
```

Reply:
I dig the last option with the example!

Reply:
🙌